packages/manager/apps/pci-instances/src/components/navigation/GoBack.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { describe, test, vi } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
+import { GoBack } from './GoBack.component';
+
+const backHref = '..';
+
+vi.mock('react-router-dom', () => ({
+  useHref: () => backHref,
+}));
+
+const getGoBackElement = () => {
+  render(<GoBack />);
+  return screen.getByText('pci_instances_common_go_back');
+};
+
+describe('Considering the GoBack component', () => {
+  test('Should be rendered with correct class and href attribute', () => {
+    const goBackElement = getGoBackElement();
+    expect(goBackElement).toBeInTheDocument();
+    expect(goBackElement).toHaveClass('mt-12 mb-3');
```

Comment:
that's is great but if you change the class mb-4 you need to fix the test ...

Reply:
Yes,
I hope class will never change 😆